### DISCUSSION TITLE
Why are Flatpak apps not following the system theme?

### DISCUSSION THREAD

**Discussion by vdawg-git** [prasanthrangan/hyprdots#39] [Q&A]
First of all: Thank you for these awesome dots! 
I am new to Linux and they have helped me a lot.

One question: Why are Flatpak apps not following the system theme?
I searched for possible causes, but did not find a clear answer.

**Comment by prasanthrangan** [comment]
I think its due to  lack of permission to access your system, that QT apps don't follow your system theme.

**Reply by vdawg-git** [reply]
Yes, I found a solution.
This gives flatpak apps the permission to access the theme: 
```bash
# Allow Flatpak apps to be styled
sudo flatpak override --filesystem=$HOME/.themes
sudo flatpak override --filesystem=$HOME/.icons
# And apply the theme settings
sudo flatpak override --env=GTK_THEME=THEME_NAME
sudo flatpak override --env=ICON_THEME=ICON_THEME_NAME
```
And this: https://github.com/refi64/stylepak
also installs the local theme as flatpak, but the apps did not change.
( Maybe both methods together worked for me )

**Reply by prasanthrangan** [reply]
Thanks, but this only works only for GTK apps, the QT apps does not seem to follow it...
I also tried installing kvantum based on [this](https://github.com/tsujan/Kvantum/issues/415), but had no luck  :(
